activerecord/lib/active_record/enum.rb

```diff
@@ -75,6 +75,19 @@ module ActiveRecord
   #
   #   Conversation.where("status <> ?", Conversation.statuses[:archived])
   #
+  # You can use <tt>:prefix</tt> option then you need to define multiple enums
+  # with same values. If prefix is <tt>true</tt>, the methods are prefixed with
+  # the name of the enum.
+  #
+  #   class Invoice < ActiveRecord::Base
+  #     enum({verification: [:done, :fail]}, prefix: true)
```

Comment:
I like prefix better. It groups them all together. I'd just have gone with visible_to, so it's `visible_to_admin?`. But I think essentially that you should just be able to choose. `enum visible_to: %i( admin user everybody ), suffix: 'visible'` would produce `admin_visible?`.